runtime/languages/assemblyscript/testdata/package-lock.json

```diff
@@ -18,11 +18,10 @@
     },
     "../../../../sdk/assemblyscript/src": {
       "name": "@hypermode/modus-sdk-as",
-      "version": "0.13.0-alpha1",
       "license": "Apache-2.0",
       "dependencies": {
         "@assemblyscript/wasi-shim": "^0.1.0",
-        "json-as": "^0.9.21",
+        "json-as": "^0.9.23",
```

Comment:
this version is broken.  I just pinned us to 0.9.21 in my last pr while Jairus fixes it.  I think you can just revert any change to this file.